README.md

```diff
@@ -20,95 +24,252 @@ var marked = transformer(require('jstransformer-marked'));
 
 var options = {};
 var res = marked.render('Some **markdown**', options);
-// => {body: 'Some <strong>markdown</strong>', dependencies: []}
+//=> {body: 'Some <strong>markdown</strong>', dependencies: []}
 ```
 
 This gives the same API regardless of the jstransformer passed in.
 
 ## API
+A transformer, once normalised using this module, will implement the following methods. Note that if the underlying transformer cannot be used to implement the functionality, it may ultimately just throw an error.
 
```

Comment:
Before, there would be a newline and empty line after headings to make reading the markdown easier. Any reason for this change?